README.md

```diff
@@ -159,11 +157,11 @@ AIConfig helps unwind complexity by separating prompts, model parameters, and mo
 
 ## Features
 
-- **Prompts as Configs**: a [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt chains and model settings in source control.
+- **Prompts as Configs**: [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt and model settings in source control.
```

Comment:
Should this be 'store prompts'? As-is, it sort of sounds like just storing prompt settings even though we store the literal prompts themselves